src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -262,7 +265,7 @@ export class ResultLayout extends Component {
   }
 
   private updateSelectorAppearance() {
-    if (this.shouldShowSelector()) {
+    if (this.shouldShowSelector() && !this.hasNoResults) {
```

Comment:
I would instead add this to the shouldShowSelector method, instead of adding an additional check here.